### PR TITLE
Poc-java naming conventions

### PR DESCRIPTION
**Pull request by SuduguTejasvi**

### CODE REVIEW

File: Calculator.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+public class Calculator {
+    public static void main(String[] args) {
+        Scanner sc=new Scanner(System.in);
+        System.out.println("Enter operation you want to perform");
+        String mode=sc.nextLine();
+        System.out.println(("Enter two numbers"));
+        int num1=sc.nextInt();
+        int num2=sc.nextInt();
+        int num3=0;
+        if(mode.equalsIgnoreCase("sum")) {
```
Comment (MajaharZemoso) [inline Calculator.java:18]: Try to use switch here

File: BestPracticesExample.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    String orderId;
+    String itemCode;
+    int quantity;
+}
+
+public class BestPracticesExample {
+
+    // Function with one parameter that checks whether a number is even or odd
+    static void isEven(int n) {
+        if (n % 2 == 0) {
```
Comment (MajaharZemoso) [inline BestPracticesExample.java:20]: add good variable name

File: Calculator.java (hunk 1 above)
Comment (SuduguTejasvi) [reply Calculator.java]: addressed it

File: BestPracticesExample.java (hunk 2 above)
Comment (SuduguTejasvi) [reply BestPracticesExample.java]: addressed it

**Review by MajaharZemoso — APPROVED** [review]

**Pull request merged**